.teamcity/settings.kts

```diff
@@ -166,7 +166,8 @@ object RunAllUnitTests : BuildType({
 			dockerRunParameters = "-u %env.UID%"
 		}
 		script {
-			name = "Code hygiene"
+			name = "Prevent uncommited changes"
+			executionMode = BuildStep.ExecutionMode.RUN_ON_FAILURE
```

Comment:
See https://teamcity.jetbrains.com/app/dsl-documentation/jetbrains.build-server.configs.kotlin.v2018_1/-build-step/-execution-mode/-r-u-n_-o-n_-f-a-i-l-u-r-e.html and https://www.jetbrains.com/help/teamcity/2020.1/configuring-build-steps.html#Execution+policy